hack/test/e2e-aws.sh

```diff
@@ -78,8 +78,7 @@ function setup {
   ## TODO: update to talos-systems once merged
```

Comment:
I wonder what this comment means

Reply:
I guess it's outdated as the templates lived under someone's else account